src/components/Dashboard/PendingFiling.vue

```diff
@@ -20,13 +20,15 @@
 </template>
 
 <script lang="ts">
-import { Component, Vue, Prop } from 'vue-property-decorator'
+import { Component, Mixins, Prop } from 'vue-property-decorator'
 import { ContactInfo } from '@/components/common'
+import { CommonMixin } from '@/mixins'
 
 @Component({
+  mixins: [CommonMixin],
```

Comment:
Same as above, i don't recall having to do this in the other UI's. Curious what is different here.

Reply:
It's not required for correct linting, building and testing.